src/tokens/TokenDetails.test.tsx

```diff
@@ -178,21 +178,22 @@ describe('TokenDetails', () => {
     expect(queryByText('tokenDetails.priceUnavailable')).toBeFalsy()
   })
 
-  it('renders chart if token is native (celo) using firebase', () => {
+  it('renders chart and news feed if token is native (celo) using firebase', () => {
```

Comment:
can we include this in the test for chart using blockchain API or just make a separate test? This test will get removed when we clean up the feature gate for price chart, and we may accidentally miss carrying this over.